CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.1] - 2020-06-26
+
+### Fixed
+
+- Bump Node.js dependencies.
+- GitHub app prefilled URL form using local team-chat.jpg.
```

Comment:
Given this seems like a a minor change, just in terms of _is the dojo following SemVer strictly_, then perhaps it should be bumped 1.3?

Reply:
Ah Adam thanks for reviewing this long-standing PR.
I use the patch digit when bumping external dependencies without code change in this repository.
The second bullet is more cryptic. Actually it refers to the new encoded URL in docs/bot-setup.md where I change the location of the image associated with the github app. It's a change for the location I should have used in first place ;o)